pom.xml

```diff
@@ -311,6 +312,1093 @@
+        <gravitee-reporter-tcp.version>2.3.1</gravitee-reporter-tcp.version>
+        <!--    Version of policy-ratelimit is also used for policy-quota, policy-spikearrest and gateway-services-ratelimit    -->
+        <!--    <gravitee-gateway-services-ratelimit.version>1.15.0</gravitee-gateway-services-ratelimit.version>    -->
+        <gravitee-tracer-jaeger.version>3.0.1</gravitee-tracer-jaeger.version>
+
+        <!-- Versions of the plugins for the full distribution on dev environment-->
+        <!-- Management API & Gateway -->
+        <!-- Community plugins -->
+        <gravitee-connector-kafka.version>1.1.1</gravitee-connector-kafka.version>
+        <gravitee-policy-aws-lambda.version>1.2.0</gravitee-policy-aws-lambda.version>
+        <gravitee-policy-circuit-breaker.version>1.1.5</gravitee-policy-circuit-breaker.version>
+        <gravitee-policy-geoip-filtering.version>2.0.0</gravitee-policy-geoip-filtering.version>
+        <gravitee-policy-javascript.version>1.3.3</gravitee-policy-javascript.version>
+        <gravitee-resource-auth-provider-http.version>1.3.2</gravitee-resource-auth-provider-http.version>
+        <gravitee-resource-auth-provider-inline.version>1.3.0</gravitee-resource-auth-provider-inline.version>
+        <gravitee-resource-auth-provider-ldap.version>1.3.0</gravitee-resource-auth-provider-ldap.version>
+        <gravitee-resource-cache-redis.version>1.5.0</gravitee-resource-cache-redis.version>
+        <gravitee-resource-oauth2-provider-keycloak.version>2.0.0</gravitee-resource-oauth2-provider-keycloak.version>
+        <gravitee-service-geoip.version>3.0.0</gravitee-service-geoip.version>
+        <gravitee-secretprovider-kubernetes.version>1.0.1</gravitee-secretprovider-kubernetes.version>
+
+        <!-- Enterprise plugins -->
+        <gravitee-entrypoint-http-get.version>1.1.0</gravitee-entrypoint-http-get.version>
+        <gravitee-entrypoint-http-post.version>1.1.0</gravitee-entrypoint-http-post.version>
+        <gravitee-entrypoint-sse.version>4.0.2</gravitee-entrypoint-sse.version>
+        <gravitee-entrypoint-webhook.version>2.3.1</gravitee-entrypoint-webhook.version>
+        <gravitee-entrypoint-websocket.version>1.0.4</gravitee-entrypoint-websocket.version>
+        <gravitee-endpoint-kafka.version>2.10.2</gravitee-endpoint-kafka.version>
+        <gravitee-endpoint-mqtt5.version>2.2.0</gravitee-endpoint-mqtt5.version>
+        <gravitee-endpoint-rabbitmq.version>1.3.1</gravitee-endpoint-rabbitmq.version>
+        <gravitee-endpoint-solace.version>1.1.0</gravitee-endpoint-solace.version>
+        <gravitee-resource-schema-registry-confluent.version>2.0.0</gravitee-resource-schema-registry-confluent.version>
+        <gravitee-reactor-message.version>2.0.0</gravitee-reactor-message.version>
+        <gravitee-repository-bridge.version>4.2.2</gravitee-repository-bridge.version>
+        <gravitee-secretprovider-hc-vault.version>1.0.0</gravitee-secretprovider-hc-vault.version>
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Config Generic   
**Description:** Generic Config Credential   
**Severity:** HIGH   
**Match:**         <gravitee-secretprovider-hc-vault.version>*****</gravitee-secretprovider-hc-vault.version>  
[This comment was created by Aqua Pipeline]